oauth2client/client.py

```diff
@@ -279,8 +280,7 @@ def new_from_json(cls, s):
       An instance of the subclass of Credentials that was serialized with
       to_json().
```

Comment:
byte string or unicode string. `_from_bytes` will take something of `six.text_type` and just leave it as-is.